src/modules/systemlib/circuit_breaker_params.c

```diff
@@ -106,7 +105,7 @@ PX4_PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK);
  * @max 121212
  * @group Circuit Breaker
  */
-PX4_PARAM_DEFINE_INT32(CBRK_FLIGHTTERM);
+PARAM_DEFINE_INT32(CBRK_FLIGHTTERM, 121212);
```

Comment:
Not used in SITL? Not sure why the circuit breakers have PX4 param defines instead of the normal ones